src/pages/DotaAPI/DotaLandingPage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import HeroApi from "../DotaAPI/HeroApi";
+import ItemApi from "../YC_Liew/Dota2";
+import MatchAPI from "../ChokKahYang/MatchAPI";
+import BasePage from "../basePage/basePage";
+import DotaTeam from "../Timothy/proteams";
+export default class DotaApi extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <BasePage title="Dota 2">
+        <BasePage title="Match (KY Chok)">
```

Comment:
what you need here is heading, basePage is use to act as a wrapper in each page

Reply:
ky resolved this on his pc and has pushed